storage/lib/storage.ex

```diff
@@ -0,0 +1,31 @@
+defmodule GoogleApi.Storage.Samples do
+  require Tesla
+
+  @moduledoc """
+  Documentation for GoogleApi.Storage.Samples.
+  """
+
+  @doc """
+  List storage buckets for a project.
+
+  ## Examples
+
+      iex> GoogleApi.Storage.Samples.list_buckets("your_project_id")
+      your_project_bucket1
+      your_project_bucket2
+
+  """
+  def list_buckets(project_id) do
+    {:ok, token} = Goth.Token.for_scope("https://www.googleapis.com/auth/cloud-platform")
```

Comment:
Probably don't need this scope which is basically all permissions.

Reply:
I think this scope is fine. It's pretty common practice to use it (especially since scopes are superseded by IAM roles for limiting access.

It's also, I think, good practice to use it for Goth, since Goth will then reuse tokens across all APIs that use this scope.

Reply:
Oh, I meant specifically the `"https://www.googleapis.com/auth/cloud-platform"` scope versus `"https://www.googleapis.com/auth/devstorage.full_control"`

Reply:
Yes, I meant I think it's fine to use `https://www.googleapis.com/auth/cloud-platform`.

Reply:
`devstorage` is a confusing scope name, it sounds like it's not meant for production. Not sure why we went with that name, but to avoid confusion is why I went with `cloud-platform`